src/Console/ConsoleOptionParser.php

```diff
@@ -589,6 +592,39 @@ public function addSubcommand($name, array $options = [])
     }
 
     /**
+     * @param array $options Options
+     * @return array
+     */
+    protected function _mergeSubcommandHelpToParserDescription($options)
+    {
+        if (!$options['help']) {
+            return $options;
+        }
+
+        if ($options['parser'] && is_object($options['parser'])) {
```

Comment:
We could check on the interface as well, I just assumed that if it is an object it must be the expected one.